### PR TITLE
Next satellites to pass for station view

### PR DESCRIPTION
**Pull request by comzeradd**
Rebased from #179.

### CODE REVIEW

**Comment by cshields** [comment]
+1

File: network/base/views.py [hunk 1]
```diff
@@ -306,10 +308,79 @@ def station_view(request, id):
     form = StationForm(instance=station)
     antennas = Antenna.objects.all()
 
+    try:
+        satellites = Satellite.objects.filter(transmitters__alive=True).distinct()
+    except:
+        pass  # we won't have any next passes to display
```
Comment (comzeradd) [inline network/base/views.py]: I think `filter()` fails silently when empty, but this try/except seems weird to me (see bellow).

File: network/base/views.py [hunk 2]
```diff
@@ -306,10 +308,79 @@ def station_view(request, id):
...
+    # calculate upcoming passes for the station
+    observer = ephem.Observer()
+    observer.lon = str(station.lng)
+    observer.lat = str(station.lat)
+    observer.elevation = station.alt
+
+    nextpasses = []
+    passid = 0
+
+    for satellite in satellites:
```
Comment (comzeradd) [inline network/base/views.py]: So what if satellites has return empty?

**Comment by cshields** [comment]
> So what if satellites has return empty?

I tested this, it does not cause any errors but the table just shows empty (as expected).  Should this case be checked for and some other message displayed?  The scenario I can think of this happening in is an uninitialized network install, otherwise there should be a populated table with satellites and transmitters from the db site, always giving something for this variable (whether or not any of them will pass the ground station is another question)

**Comment by comzeradd** [comment]
I'll test it locally and I think it's good to be merged. Some minor UI changes (like opt-in button to see the next passes) or housekeeping (move calculations to another module to be used by both next passes and observations scheduling) may happen on another PR.

**Pull request merged**